iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergFilterFactory.java

```diff
@@ -138,21 +138,6 @@ public void testBetweenOperand() {
     assertEquals(actual.right().op(), expected.right().op());
   }
 
-  @Test
-  public void testUnsupportedBetweenOperandEmptyLeaves() {
-    SearchArgument.Builder builder = SearchArgumentFactory.newBuilder();
-    final SearchArgument arg =
-        new MockSearchArgument(
```

Comment:
Maybe this mock class also need to be removed.
https://github.com/apache/hive/blob/36ce858163a19e29eafe4a8d3307191bc28fc175/iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergFilterFactory.java#L282

Reply:
removed